src/suparnaNJune20/Assignment_31/InputNumberExceptionHandling.java

```diff
@@ -0,0 +1,34 @@
+package suparnaNJune20.Assignment_31;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class InputNumberExceptionHandling {
+	static int sum = 0, cnt = 0;
+	public void sumOfNumber(int inputCnt)
+	{
+		
+		
+		Scanner scanner = new Scanner(System.in);
+		for ( cnt = 1; cnt <= inputCnt; cnt++) {
+			
+			try {
+				System.out.println("Enter "+ cnt +" input Number ");
+				sum += scanner.nextInt();
+				
+			} catch (InputMismatchException inputMismatchException) {
+				System.out.println("Enter Number input not a string  ");
+				scanner.next();
+				
+			}
+		}
+		System.out.println("Sum = "+sum);
```

Comment:
We need to display all the given input numbers and the sum in single line.
eg: "Values are 10, 30, 40 and total is 80."

Reply:
Changes Done.